deployment/settings/services/imagetag.setting

```diff
@@ -1,4 +1,4 @@
 MINDARO_DEVHOSTAGENT_TAG=1.1.0
-MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG=1.1.0
+MINDARO_DEVHOSTAGENT_RESTORATIONJOB_TAG=1.2.0
```

Comment:
Updating this based off this read me, so that newer version of restoration job will be pulled. - https://github.com/Azure/Bridge-To-Kubernetes/tree/main/src/devhostagent#versioning